eth/beacon/types/validator_records.py

```diff
@@ -41,12 +45,12 @@ class ValidatorRecord(rlp.Serializable):
     ]
 
     def __init__(self,
-                 pubkey: int,
+                 pubkey: BLSPubkey,
                  withdrawal_credentials: Hash32,
                  randao_commitment: Hash32,
-                 randao_layers: int,
-                 status: int,
-                 latest_status_change_slot: int,
+                 randao_layers: SlotNumber,
```

Comment:
Ah, `randao_layers` is the "5 slots" thing.
```suggestion
                 randao_layers: int,
```